### PR TITLE
add another matrix-free laplace GMG unit test

### PR DESCRIPTION
**Pull request by davydden**
p.s. the output is copy-paste from `parallel_multigrid.cc` test.

### CODE REVIEW

**Comment by davydden** [comment]
I also renamed output files to remove `with_trilinos=true`.

**Comment by tjhei** [comment]
:+1:

File: tests/matrix_free/parallel_multigrid_02.cc [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+#include <deal.II/matrix_free/matrix_free.h>
+#include <deal.II/matrix_free/fe_evaluation.h>
+#include <deal.II/matrix_free/operators.h>
+
+std::ofstream logfile("output");
+
+
+template <typename MatrixType>
+class MGTransferPrebuiltMF : public MGTransferPrebuilt<LinearAlgebra::distributed::Vector<double> >
+{
```
Comment (kronbichler) [inline tests/matrix_free/parallel_multigrid_02.cc]: But this class needs Trilinos (at least when run with MPI). Have a look at the top of `include/deal.II/multigrid/mg_transfer.h`, class `MatrixSelector`. If there is no Trilinos, it uses SparseMatrix but that will not work with more than one MPI rank. In other words, did you actually check that `parallel_multigrid` works with MPI but without Trilinos?

Comment (tjhei) [inline tests/matrix_free/parallel_multigrid_02.cc]: @davydden ?

Comment (davydden) [inline tests/matrix_free/parallel_multigrid_02.cc]: will fix it...

**Pull request merged**